docs/setting-up-a-new-node.md

```diff
@@ -143,6 +143,17 @@ You can get the binary in several ways:
 - Compile from source code - [instruction](../README.md);
 - Get `tar` archive with the binary compiled for Ubuntu 20.04 in [releases](https://github.com/cheqd/cheqd-node/releases);
 
+### Setting up `cheqd-noded` binary as a service
+
+It's highly recommended running node under the service, for example `systemd` system can be used for it. 
+Our debian package uses [postinst](https://github.com/cheqd/cheqd-node/blob/main/build_tools/postinst) script for setting up our binary as a service and it was designed for using outside deb package too. 
+For setting up your binary as a service on ubuntu just run (it requires root privileges):
+```
+# bash postinst <path/to/cheqd-noded/binary> 
+```
+and this script will add service file and prepare all needed directories for `configs/keys` and `data`.
+Please make sure, it assumes to create and use `cheqd` user for service and all the manipulations with `cheqd-noded` should be run under the `cheqd` user in the future.
```

Comment:
I'm not sure what this comment means...does it mean that postint  **will** create a user called `cheqd`, or should the user previously have created a user called `cheqd`?

Either way, are there any parameters that need to be passed to `postinst` so that it configures directories or to expect/use the `cheqd` user correctly?

Reply:
@anikitinDSR Hey could you please take a look at the above and resolve? I also made a commit with some edits, so you might have to sync with that commit first.

Reply:
Agreed, it's not the correct phrase. I meant that this script will create user `cheqd`  cause it's needed for systemd to start with separate from another users environment.